tests/opensource-suites/charts/preview/widget-width.test.ts

```diff
@@ -0,0 +1,26 @@
+import {Page} from '@playwright/test';
+
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+import {openTestPage, slct} from '../../../utils';
+import {ChartKitTableQa, PreviewQa} from '../../../../src/shared';
+import {PreviewUrls} from '../../../constants/test-entities/charts';
+
+datalensTest.describe('Charts Preview', () => {
+    datalensTest.beforeEach(async ({page}: {page: Page}) => {
+        await openTestPage(page, PreviewUrls.FlatTableWithOneColumn);
+
+        const initHeight = page.viewportSize()?.height;
+        if (initHeight) {
+            page.setViewportSize({width: 800, height: initHeight});
+        }
+    });
+
+    datalensTest('table widget must be the same', async ({page}) => {
```

Comment:
Please write a comment - what should be checked. This will be useful in case of updating screenshots - what exactly you need to pay attention to